iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -213,6 +215,7 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
         organization_customizations = [
```

Comment:
None of them will be applied except for `resolve_pull_specs` which is under `registry_replacements` (pre-existing code) and has to be done irrespective of the customization org. This is to avoid a huge else block where we call a bunch of functions and avoid code duplication.

Reply:
More specifically, this => https://github.com/release-engineering/iib/commit/30525746d11bfe833ce78b3516e2960ed17514da#diff-1f7f19fb78fb596298715dcd50d317bee99dc1e246c61918856ee9c716f0eff8R242